doc/sphinx/changes.rst

```diff
@@ -865,8 +865,8 @@ Specification language, type inference
   If the value is missing, the default is :n:`yes`.  The old syntax is still
   supported, but produces the ``deprecated-attribute-syntax`` warning.
 
-  Deprecated attributes are :attr:`universes(monomorphic)`,
-  :attr:`universes(notemplate)` and :attr:`universes(noncumulative)`, which are
+  Deprecated attributes are ``universes(monomorphic)``,
```

Comment:
Would it be better to have a "deprecated attributes" page so these can still be referenced? That way these can also be removed easily in the future.

Also do we even need to mention them anymore?

cc @Zimmi48 @jfehrle

Reply:
> Would it be better to have a "deprecated attributes" page so these can still be referenced? That way these can also be removed easily in the future.

I really don't know, I don't feel relevant enough about the manual to take such a decision.

> Also do we even need to mention them anymore?

This is the changelog, and IMHO it shouldn't be edited (or minimally, but at least nothing should be removed).

Reply:
We have always done it like this and IMHO this is fine.